src/components/volunteer-enroll/SubmitCriminalCheck.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+    OrderedList,
+    ListItem,
+    Flex,
+    Text,
+    Checkbox,
+    Box,
+    Button,
+    Link,
+} from "@chakra-ui/react";
+import { ClassCardInfo } from "@models/Class";
+import colourTheme from "@styles/colours";
+import React, { useState } from "react";
+
+type SubmitCriminalCheckFormProps = {
+    styleProps?: Record<string, unknown>;
+    onNext: () => void;
+    classInfo: ClassCardInfo;
+};
+/**
+ * Media release page within the class registration process
+ * @returns a page component explaining SDC's media release policy and offering an option to accept it
+ */
+export const SubmitCriminalCheckForm: React.FC<SubmitCriminalCheckFormProps> =
+    ({ onNext, classInfo }): JSX.Element => {
+        // Next button is disabled by default, activates criminal checkbox is checked
+        const [criminalCheck, setCriminalCheck] = useState<boolean>(false);
+
+        return (
+            <>
+                <Flex justifyContent="space-between"></Flex>
+                <Box>
+                    <Text
+                        align="left"
+                        mt="35px"
+                        fontWeight="700"
+                        fontSize="36px"
+                    >
+                        Submit Background Check
+                    </Text>
+                </Box>
+                <Box>
+                    <Text
+                        pb="5px"
+                        align="left"
+                        fontWeight="700"
+                        mt="50px"
+                        fontSize="22px"
+                    >
+                        You have not uploaded a Criminal Record Check yet!
```

Comment:
Please replace this with provided translation, by using `const {t} = useTranslation("form")` - note that you will need to include this json in `getServerSideProps` in the base page - `...(await serverSideTranslations(context.locale, ["common", "form"])),`
```suggestion
                        t("bgc.missing")
```